src/src/pages/capabilities/NewCapabilityDialog.js

```diff
@@ -1,8 +1,18 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import { Button, ButtonStack } from "@dfds-ui/react-components";
 import { SideSheet, SideSheetContent } from "@dfds-ui/react-components";
 import { Tooltip, TextField } from "@dfds-ui/react-components";
 import styles from "./capabilities.module.css";
+import { getUsers } from "GraphApiClient";
+import DropDownMenu from "components/DropDownMenu";
+
+function TextWrapper() {
+  return (
+    <>
+      <div className=""></div>
+    </>
+  );
+}
```

Comment:
is this really needed?

Reply:
Addressed in https://github.com/dfds/selfservice-portal/pull/133/commits/cb4b6bfc44c8d7dff3c4ea8e1c66e2dc416c35bd